tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -41,7 +41,8 @@ def tearDown(self, duthost, localhost, pdu_controller):
             # Wait until all critical processes are healthy.
             wait_critical_processes(duthost)
 
-    def test_memory_exhaustion(self, duthost, localhost):
+    def test_memory_exhaustion(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost):
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
```

Comment:
Similar fix is needed for autouse fixture:
 tearDown(self, duthost, localhost, pdu_controller):

Reply:
Fixed